test/SConscript

```diff
@@ -47,10 +41,16 @@ env_rm_png  = env.Clone(COPY = 'rm -f *.png')
 enzo_bin = bin_path + '/enzo-p'
 
 # h5topng
+
+#----------------------------------------------------------------------
+SConscript('Balance/SConscript')
+#----------------------------------------------------------------------
+
 #----------------------------------------------------------------------
 # ARRAY COMPONENT         
 #----------------------------------------------------------------------
-env.RunSerial('test_CelloArray.unit',bin_path + '/test_CelloArray')
+#env.RunSerial('test_CelloArray.unit',bin_path + '/test_CelloArray')
```

Comment:
This test got added during PR #62. It must have fallen through the cracks during the merging process.

What do we need to do to get this test running? Do I just need to setup an `ArrayComponent/SConscript` file and move this line to that file?

Reply:
Yes - exactly. I think you can follow what is done in MethodPpm  - that appears to do something similar. Maybe once this PR gets pulled in you can issue that PR?

Reply:
Sure, will do.